### DISCUSSION TITLE
Not able to use the benefits of Github Student Pack after getting accepted

### DISCUSSION THREAD

**Discussion by Vidit02** [community/community#58438] [GitHub Education] [answered]
### Hi everyone,

Hello everyone, 
I am not able to use my github student benefits after it's been 3 days of getting accepted.
I am attaching the screenshot too please look into it and solve my error.
thanks a lot!!!
[Uploading Screenshot 2023-06-19 at 2.01.08 PM.png…]()

**Comment by the-officialjosh** [comment, marked as answer]
If you have already received the Pro status as part of the GitHub student benefits, please be patient as additional features will be gradually rolled out to you. It's worth noting that there might be delays, as mentioned on their webpage. To gather more information and stay updated on any potential delays, I recommend visiting the following page: [Education GitHub Pack Offers](https://education.github.com/pack/offers). The provided page should provide you with further details and clarification regarding the rollout of features.

**Comment by samus-aran** [comment]
Hi @Vidit02 ! We hope you found what you were looking for. ✨

We're locking this discussion because it has not had recent activity and/or other members have asked for more information to assist you but received no response. Thank you for helping us maintain a productive and tidy community for all our members.
